docs/changelogs/v0.26.md

```diff
@@ -18,10 +18,10 @@
 
 Several deprecated commands have been removed:
 
-- `ipfs urlstore` deprecated in [April 2019, Kubo 0.4.21](https://github.com/ipfs/kubo/commit/8beaee63b3fa634c59b85179286ad3873921a535)
+- `ipfs urlstore` deprecated in [April 2019, Kubo 0.4.21](https://github.com/ipfs/kubo/commit/8beaee63b3fa634c59b85179286ad3873921a535), use `ipfs add -q --nocopy --cid-version=1 {file}` instead.
```

Comment:
nit: should this be `{url}` instead of `{file}` ?  
(urlstore != filestore, these are separate experiments)